js/src/admin/utils/getCategorizedExtensions.js

```diff
@@ -0,0 +1,29 @@
+export default function getCategorizedExtensions() {
+  let extensions = {};
+
+  Object.keys(app.data.extensions).map((id) => {
+    const extension = app.data.extensions[id];
+    let category = extension.extra['flarum-extension'].category;
+
+    if (!extension.extra['flarum-extension'].category) {
```

Comment:
Is this needed? If it's null, null wouldn't be in `app.extensionCategories`, so wouldn't it go to `other` regardless?